mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
+      if (!deletedOrder) {
+        return res.status(404).json({ error: "No such order found" });
+      }
+      res.json({ success: "Order has been deleted" });
+    })
+    .catch(error => {
```

Comment:
The catch block should handle the error in a more informative way. Instead of just returning the error, consider logging the error or sending a more descriptive error message to the client. For example, you could log the error using a logging library like Winston, and then return a 500 error to the client with a generic error message.